pkg/certificateauthority/handler.go

```diff
@@ -0,0 +1,62 @@
+package certificateauthority
+
+import (
+	"context"
+	"os"
+
+	"github.com/sirupsen/logrus"
+	corev1 "k8s.io/api/core/v1"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+// NewHandler returns a new Handler.
+func NewHandler(client client.Client) Handler {
+	return &configmaphandler{
+		client: client,
+	}
+}
+
+// Handler is the interface that wraps the Handle method
+//
+// Handle handles a new event associated with OperatorSource type.
+//
+// ctx represents the parent context.
+// event encapsulates the event fired by operator sdk.
+type Handler interface {
+	Handle(ctx context.Context, operatorSource *corev1.ConfigMap) error
+}
+
+// configmapHandler implements the Handler interface
```

Comment:
I will change `configmapHandler` to `configmaphandler` once reviews are finished.